modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -57,3 +58,143 @@ When using Salt or {productname} (either via UI or API) on any {caasp} nodes:
+
+
+
+=== Manage Clusters
+
+To manage a cluster in {productname}, add the cluster in the {webui}.
+
+
+
+.Procedure: Adding an Existing Cluster
+. In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[FIXME].
+. Follow the prompts to provide information about your cluster, including the cluster type, and select the management node to associate.
+. Type the path to the ``skuba`` configuration file for the cluster.
+// For example?
+. Type the path passwordless SSH key you want to use, or to the ``ssh-agent`` socket.
+. Type a name, label, and description for the cluster.
+. Click btn:[FIXME].
+
+
+For each cluster you manage with {productname}, a corresponding system group is created.
+By default, the system group is called ``Cluster <cluster_name>``.
+Refresh the system group to update the list of nodes.
+Only nodes known to {productname} are shown.
+
+
+You can remove clusters from {productname} by navigating to menu:Clusters[Overview], unchecking the cluster to be deleted, and clicking btn:[Delete Cluster].
+
+
+[IMPORTANT]
+====
+Deleting a cluster removes the cluster from {productname}, it does not delete the cluster nodes.
+Workloads running on the cluster will continue uninterrupted.
+====
+
+
+
+=== Manage Nodes
+
+When you have the cluster created in {productname}, you can manage nodes within the cluster.
+
+Before you add a new node to the cluster, check the management node can access the node you want to add using passwordless SSH, or the ``ssh-agent`` socket you are forwarding.
+
+You also need to ensure that the node you want to add is registered to {productname}, and has a {caasp} channel assigned.
+
+
+.Procedure: Adding a Node to a Cluster
+. In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[Join Node].
+. Select the node to add from a list of available nodes.
+The available nodes includes only nodes that are registered to {productname}, are not management nodes, and are not currently part of any cluster.
+. Follow the prompts to enter the {caasp} parameters for the node to be added.
+. OPTIONAL: Specify a custom ``ssh-agent`` socket that is valid only for the nodes that are being added.
+. Click btn:[Save] to schedule an action to add the node.
+During this action, {productname} prepares the node for joining by disabling swap, then joins the node to the cluster.
+
+
+
+.Procedure: Removing a Node from a Cluster
+. In the {productname} {webui}, navigate to menu:Clusters[Overview], uncheck the nodes to remove, and click btn:[Remove Node].
+. Follow the prompts to define the parameters for the node to be removed.
+. OPTIONAL: Specify a custom ``ssh-agent`` socket that is valid only for the nodes that are being removed.
+. Click btn:[Save] to schedule an action to remove the node.
+
+For more information about node removal, see https://documentation.suse.com/suse-caasp/4/single-html/caasp-admin/\#\_permanent_removal
```

Comment:
```suggestion
For more information about node removal, see https://documentation.suse.com/suse-caasp/4/single-html/caasp-admin/#_permanent_removal.
```
It would be weird if you actually must escape these characters

Reply:
Oh, I escaped them because it was breaking my highlighting in my editor. Forgot to remove them, clearly!